conf/modules/orange_avoider.xml

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE module SYSTEM "module.dtd">
+
+<module name="orange_avoider">
+  <doc>
+    <description>Example on how to use the colours detected to avoid orange pole in the cyberzoo</description>
```

Comment:
first line of description should be the title/name of the module, extended description after that.